locales/ruRU.lua

```diff
@@ -305,8 +305,8 @@ local L = app.L;
 		L.MUSIC_ROLLS_SELFIE_FILTERS_CHECKBOX = "|T"..app.asset("Expansion_WOD")..":0|t |cffADD8E6Мелодии / Фильтры селфи";
 		L.MUSIC_ROLLS_SELFIE_FILTERS_CHECKBOX_TOOLTIP = "Включите для отслеживания мелодий и фильтров селфи.\n\nВы можете использовать игрушку Переносной музыкальный проигрыватель, чтобы воспроизводить внутриигровую музыку, и Камеру СЕЛФИ 2.0, чтобы собирать селфи для фотографий в определённых местах!";
 		L.QUESTS_CHECKBOX_TOOLTIP = "Включите для отслеживания обычных Заданий.\n\nПри помощи правого клика на любое задание в списках Вы можете открыть полную цепочку заданий, чтобы отобразить Ваш прогресс и любые предшествующие Задания.\n\nПримечание: Задания не отслеживаются перманентно из-за природы отслеживания Ежедневных, Еженедельных, Ежегодных и Локальных заданий в базе данных Blizzard.";
-		L.QUESTS_LOCKED_CHECKBOX = "|cffADD8E6+Хлебные Крохи";
-		L.QUESTS_LOCKED_CHECKBOX_TOOLTIP = "Включите для специального отслеживания выполнения Заданий-\"Хлебных Крох\".\n\nХлебные Крохи технически 'опциональные', поскольку служат для того, чтобы привести игрока к другому Заданию, и становятся недоступными, если не выполнены до завершения следующего за ними Задания(ий).\nВыполнение Хлебных Крох полагается на Синхронизацию группы или отслеживание заданий для Всего Аккаунта";
+		L.QUESTS_LOCKED_CHECKBOX = "|cffADD8E6Ограниченные Задания";
+		L.QUESTS_LOCKED_CHECKBOX_TOOLTIP = "Включите для специального отслеживания выполнения Ограниченных Заданий.\n\nОграниченные Задания - это те, которые игрок не может выполнить (исходя из известных ATT Данных) при обычной игре.\n\nПолучение этих Заданий полагается на Синхронизацию группы или отслеживание заданий для Всего Аккаунта";
```

Comment:
```suggestion
		L.QUESTS_LOCKED_CHECKBOX_TOOLTIP = "Включите для специального отслеживания выполнения Ограниченных Заданий.\n\nОграниченные Задания - это те, которые игрок не может выполнить (исходя из известных ATT данных) при обычной игре.\n\nПолучение этих Заданий полагается на Синхронизацию группы или отслеживание заданий для Всего Аккаунта";
```
I don't feel too strongly about it so you can leave it like this if you want.